packages/synapse-react-client/src/mocks/msw/handlers.ts

```diff
@@ -68,6 +69,7 @@ const getHandlers = (backendOrigin: string, portalOrigin?: string) => [
   ...getResetTwoFactorAuthHandlers(backendOrigin),
   ...getMessageHandlers(backendOrigin),
   getFeatureFlagsOverride({ portalOrigin }),
+  ...getHandlersForTableQuery(backendOrigin),
```

Comment:
Always load the table query handlers, since the handlers now just reference a static data store.